app/src/main/java/bubbleheads/buy_cook/CategoriesFragment.java

```diff
@@ -0,0 +1,43 @@
+package bubbleheads.buy_cook;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+public class CategoriesFragment extends Fragment {
+    private ArrayList<Category> categoryArrayList = new ArrayList<>();
+    private RecyclerView recyclerView;
```

Comment:
you can inline all fields. For now there is no need to keep them as field. Enough to have them as local variables